iOS/chronreact/AppDelegate.m

```diff
@@ -18,51 +19,31 @@ @implementation AppDelegate
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
   [BatchPush enableAutomaticDeeplinkHandling:NO];
+  
+  // Print out the Batch installation id for testing
```

Comment:
This is for testing purposes - logs out the batch id of the specific app instance. Can use this for notification testing over the Batch API. I can give you the file I used to test (just remind me).